docs/source/development.rst

```diff
@@ -0,0 +1,104 @@
+.. _development-policy:
+
+==================
+Development policy
+==================
+
+In this section, we describe the development policy the core team of Chainer Chemistry follows.
```

Comment:
we describe the development policy that the core developers follow.

Reply:
fix